addons/account_ubl/models/account_move.py

```diff
@@ -1,29 +1,21 @@
 # -*- coding: utf-8 -*-
```

Comment:
Maybe all the edi should be prefixed in the same way to retrieve them easily
account_edi
account_edi_facturx
account_edi_ubl?